bot.py

```diff
@@ -1,23 +1,29 @@
+import discord
+import asyncio
 from discord.ext import commands
 from decouple import config
 
 
-def main():
-
+async def main():
     """This is the main method for the bot. It creates the bot,
     and loads in the commands and events from the cogs directory."""
 
-    bot = commands.Bot(command_prefix="!")
+    intents = discord.Intents.all()
```

Comment:
updated to the latest version of discord.py. Intents are now a thing, and cogs are now required to be async.